proxy/src/http/project.rs

```diff
@@ -103,6 +104,22 @@ where
         .and_then(handler::list)
 }
 
+/// `GET /discover`
+fn discover_filter<R>(
+    ctx: http::Ctx<R>,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone
+where
+    R: registry::Client + 'static,
+{
+    path("discover")
+        .and(warp::get())
+        .and(http::with_context(ctx))
+        .and(document::document(document::description(
```

Comment:
We do return a list of projects here we should describe the body in more detail. e.g.:

https://github.com/radicle-dev/radicle-upstream/blob/64371b53e20caa9f67803842f2e0136e13fff40e/proxy/src/http/project.rs#L93-L102